src/lib/style/base/_core.scss

```
@@ -1286,6 +1286,55 @@ main,
     }
   }
 
+  @mixin border-radius($radius) {
+    -webkit-border-radius: $radius;
+    -moz-border-radius: $radius;
+    -ms-border-radius: $radius;
+    border-radius: $radius;
+    -o-border-radius: $radius;
+    -khtml-border-radius: $radius;
+  }
+  
+  // Set up default bootstrap alert styles across the portals
+  .bootstrap-4-backport.alert {
```

I think design would like all portals to have matching alert styles, so let's remove css from portal side and rely on this shared definition.

---

Is this correct, or should it be this?

```suggestion
  .bootstrap-4-backport .alert {
```

If the original is correct, then this will only impact FullWidthAlert, so I think this style should be moved there (and should probably also include the .FullWidthAlert class)

If the suggestion is correct, then this will impact many components in SRC, which I would want to spot check